src/popup/index-dark.scss

```diff
@@ -0,0 +1,5 @@
+.dark {
+  @import 'styles/settings/common';
+  @import 'styles/settings/dark';
+  @import 'styles/main';
```

Comment:
I may be missing something, but why does `.dark` also contain the `common` and `main` styles? I thought the `dark` part was coming on top of what's already there 🤔

Reply:
common is just the Bootstrap Sass variables that I want to apply to both the default and the dark theme. It essentially just makes sure, we start out with the same values (Sass processing is somewhat weird).